Library/Homebrew/cmd/install.rb

```diff
@@ -145,6 +145,14 @@ def install_args
 
   def install
     args = install_args.parse
+    
+    args.named.each do |name|
+      next if File.exist?(name)
+      next if name !~ HOMEBREW_TAP_FORMULA_REGEX && name !~ HOMEBREW_CASK_TAP_CASK_REGEX
+ 
+      tap = Tap.fetch(Regexp.last_match(1), Regexp.last_match(2))
+      tap.install unless tap.installed?
```

Comment:
This seems to duplicate the `taps.reject(&:installed?).each(&:install)` from below. I think the issue is that the available formulae are being cached somewhere and then not recalculated after tapping.

Reply:
Yeah, I only did this as a quick fix on mobile.

The issue is that `Formulary` doesn't automatically install taps like `CaskLoader` does for `my/tap/cask`, so it fails to load the formula.